### PR TITLE
Fix wrong translation option for boolean type

### LINKED ISSUE #173: Boolean labels not translating with Symfony 2.7
We just upgraded from Symfony 2.3 to 2.7 and have noticed an issue with the latest version of SonataAdminBundle. The label_type_yes and label_type_no boolean value strings are not translating in the filter grid. At first we thought we needed to make some kind of configuration update in our application, but we could not find any issues.

I created a brand new Symfony 2.7 using the instructions in the cookbook, composer required composer require sonata-project/admin-bundle, and then created an entity with a single boolean field, and an admin for it. The label_type_yes and label_type_no still show up un-translated.

I am running PHP 5.5, Symfon 2.7.1 and have tried with SonataAdminBundle 2.3.3 and dev-master.

```php
<?php
namespace AppBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Bar
 *
 * @ORM\Table()
 * @ORM\Entity
 */
class Bar
{
    /**
     * @var integer
     *
     * @ORM\Column(type="integer")
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="AUTO")
     */
    private $id;

    /**
     * @var boolean
     *
     * @ORM\Column(type="boolean")
     */
    private $active;
}
```

``` php
<?php
namespace AppBundle\Admin;

use Sonata\AdminBundle\Admin\Admin;
use Sonata\AdminBundle\Datagrid\DatagridMapper;
use Sonata\AdminBundle\Datagrid\ListMapper;
use Sonata\AdminBundle\Form\FormMapper;
use Sonata\AdminBundle\Show\ShowMapper;

class BarAdmin extends Admin
{
    public function configureFormFields( FormMapper $form )
    {
        $form->add( 'active' );
    }

    public function configureListFields( ListMapper $list )
    {
        $list->add( 'active' );
    }

    public function configureDataGridFilters( DatagridMapper $filter )
    {
        $filter->add( 'active' );
    }

    public function configureShowFields( ShowMapper $show )
    {
        $show->add( 'active' );
    }
}
```

``` yml
services:
  app.admin.bar:
    class: AppBundle\Admin\BarAdmin
    tags:
      - { name: sonata.admin, manager_type: orm, group: Foo, label: Bar }
    arguments:
      - ~
      - AppBundle\Entity\Bar
      - ~
    calls:
      - [ setTranslationDomain, [AppBundle]]
```

``` html
<select id="filter_active_value" name="filter[active][value]" class="select2-offscreen" tabindex="-1" title="Active">
  <option value=""></option>
  <option value="1">label_type_yes</option>
  <option value="2">label_type_no</option>
</select>
```

I had reported this at the [SonataAdminBundle project](https://github.com/sonata-project/SonataAdminBundle/issues/3106), but they said the translation is dealt with in core.

### PR DESCRIPTION
**Pull request by Soullivaneuh**
Fixes #173.

### CODE REVIEW

File: Form/Type/BooleanType.php [hunk 1]
```diff
@@ -50,8 +50,8 @@ public function setDefaultOptions(OptionsResolverInterface $resolver)
     public function configureOptions(OptionsResolver $resolver)
     {
         $resolver->setDefaults(array(
-            'catalogue' => 'SonataCoreBundle',
-            'choices'   => array(
+            'translation_domain' => 'SonataCoreBundle',
+            'choices'            => array(
```
Comment (OskarStark) [inline Form/Type/BooleanType.php]: this indention looks weird

Comment (Soullivaneuh) [inline Form/Type/BooleanType.php]: What do you mean? It's an automatic correction of php CS.

Comment (OskarStark) [inline Form/Type/BooleanType.php]: hmm, ok

**Comment by rande** [comment]
ok to test

**Comment by Soullivaneuh** [comment]
ok to test

**Comment by Soullivaneuh** [comment]
ok to test

**Comment by Soullivaneuh** [comment]
ok to test

**Pull request merged**